lms/djangoapps/ccx/views.py

```diff
@@ -146,6 +147,9 @@ def dashboard(request, course, ccx=None):
         context['grading_policy_url'] = reverse(
             'ccx_set_grading_policy', kwargs={'course_id': ccx_locator})
 
+        with ccx_course(ccx_locator) as course:
+            context['course'] = course
```

Comment:
This code adds `'course'` to the context - but I don't see any accompanying changes to the template, nor any reference in the existing template? Why was this attribute added to the context?